libraries/AP_RangeFinder/RangeFinder.cpp

```diff
@@ -227,6 +227,14 @@ const AP_Param::GroupInfo RangeFinder::var_info[] = {
 #endif
 
 #if RANGEFINDER_MAX_INSTANCES > 2
+    // @Param: 3_ADDR
+    // @DisplayName: Bus address of third rangefinder
+    // @Description: This sets the bus address of the sensor, where applicable. Used for the LightWare I2C sensor to allow for multiple sensors on different addresses. A value of 0 disables the sensor.
+    // @Range: 0 127
+    // @Increment: 1
+    // @User: Advanced
+    AP_GROUPINFO("3_ADDR", 36, RangeFinder, _address[2], 0),
+
```

Comment:
This is nitpicking but is there any reason for putting this at the beginning and not at the end of the _if_ like the others?
We should also move the *_ADDR* and *2_ADDR* to be together with their respective parameters